src/frontend/src/test-e2e/util.ts

```diff
@@ -371,6 +412,16 @@ export async function switchToPopup(
   return await addVirtualAuthenticator(browser);
 }
 
+/**
+ * Switches to the first window of the given browser. Explicit window focusing is required when using the mobile emulation mode of chrome.
```

Comment:
Do you know why focusing is required? Seems odd that we need to explicitly focus

Reply:
Yes, the mobile emulation comes with it's own sets of weird quirks. If this is removed, the browser just gets stuck on the wrong screen. I'm not sure why the chrome developer chose this behavior.